src/hooks/login/useWalletConnectV2Login.ts

```diff
@@ -0,0 +1,249 @@
+import { useRef, useState } from 'react';
+import {
+  WalletConnectProviderV2,
+  PairingTypes
+} from '@elrondnetwork/erdjs-wallet-connect-provider';
+import { useUpdateEffect } from 'hooks/useUpdateEffect';
+import {
+  getAccountProvider,
+  setAccountProvider
+} from 'providers/accountProvider';
+import { loginAction } from 'reduxStore/commonActions';
+import { useDispatch, useSelector } from 'reduxStore/DappProviderContext';
+import { isLoggedInSelector } from 'reduxStore/selectors/loginInfoSelectors';
+import {
+  walletConnectV2RelaySelector,
+  walletConnectV2ProjectIdSelector,
+  chainIDSelector,
+  walletConnectDeepLinkSelector
+} from 'reduxStore/selectors/networkConfigSelectors';
+import {
+  setTokenLogin,
+  setTokenLoginSignature,
+  setWalletConnectLogin
+} from 'reduxStore/slices';
+import { LoginHookGenericStateType } from 'types';
+import { LoginMethodsEnum } from 'types/enums';
+import { logout } from 'utils';
+import { optionalRedirect } from 'utils/internal';
+
+interface InitWalletConnectV2Type {
+  logoutRoute: string;
+  token?: string;
+  callbackRoute?: string;
```

Comment:
Do we still need this file? Seems like a lot of logic has been moved to the above hook